app/models/preneeds/burial_form.rb

```diff
@@ -2,48 +2,89 @@
 
 require 'common/models/base'
 
+# Preneeds namespace
+# 
 module Preneeds
+  # Models a Preneeds Burial form. This class (and associationed classes) is used to store form details and to
+  # generate the XML request for submission to the EOAS service
+  #
+  # @!attribute application_status
+  #   @return [String] currently always blank
+  # @!attribute preneed_attachments
+  #   @return [Array<PreneedAttachmentHash>] documents provided by the end user
+  # @!attribute has_currently_buried
+  #   @return [String] '1' for Yes, '2' for No, '3' for Don't know
+  # @!attribute sending_application
+  #   @return [String] hard coded as 'va.gov'
+  # @!attribute sending_code
+  #   @return [String] currently always blank
+  # @!attribute sent_time
+  #   @return [Time] current time
+  # @!attribute tracking_number
+  #   @return [String] SecureRandom generated tracking number sent with submission to EOAS
+  # @!attribute applicant
+  #   @return [Preneeds::Applicant] Applicant object. Applicant is person filling out the form.
+  # @!attribute claimant
+  #   @return [Preneeds::Claimant] Claimant object. Claimant is the person applying for the
+  #     benefit (veteran or relative)
+  # @!attribute currently_buried_persons
+  #   @return [Array<Preneeds::CurrentlyBuriedPerson>] CurrentlyBuriedPerson objects representing individuals burried in
+  #     VA national cemeteries under the sponsor's eligibility
+  # @!attribute veteran
+  #   @return [Preneeds::Veteran] Veteran object.  Veteran is the person who is the owner of the benefit.
+  #
   class BurialForm < Preneeds::Base
-    attribute :application_status, String # TODO: not in schema, not in submission from frontend
+    # Preneeds Burial Form officail form id
+    #
+    FORM = '40-10007'
+
+    attribute :application_status, String, default: ''
     attribute :preneed_attachments, Array[PreneedAttachmentHash]
     attribute :has_currently_buried, String
     attribute :sending_application, String, default: 'vets.gov'
-    attribute :sending_code, String # TODO: not in schema, not in submission from frontend
+    attribute :sending_code, String, default: ''
```

Comment:
Added empty string as default instead of using `||` below (see line 87).
This attribute is not in the json schema and thus not in any submission from the front end, so it will always just be an empty string.